feg/protos/mconfig/mconfigs.proto

```diff
@@ -244,3 +244,41 @@ message S8Config {
     string pgw_address = 3;
     string apn_operator_suffix = 4;
 }
+
+message HttpServerConfig {
```

Comment:
shouldn' this be `SbiServerConfig`?

Or does this object really reprepresents a generic HTTP server?